src/sentry/event_manager.py

```diff
@@ -711,6 +762,11 @@ def _get_or_create_release_many(jobs, projects):
             date_added=release_date_added[(project_id, version)],
         )
 
+        if features.has(
+            "projects:auto-associate-commits-to-release", projects[project_id]
+        ) and _is_commit_sha(release.version):
```

Comment:
In some projects the version of the release may already include the sha as part of the version.
e.g. `backend@22.8.0+a0a73ec041d102a42c46db401b445eeb7f91e485` (related screenshot [1])

We may be able to extract the sha from there for those projects.
[1]
<img width="501" alt="image" src="https://user-images.githubusercontent.com/44410/180061120-259c387f-9847-4248-a4df-e312be45cff2.png">

Reply:
From what I saw in the tech spec, we should expect the release name to just be a checksum in workflow 2.0. This case might still be worth looking into.

@NisanthanNanthakumar Is this something we should deal with in this initial version?